cli.md

```diff
@@ -308,7 +312,7 @@ Usage: guardian workflows plan-status-comment [options] <pull_request_number>
 
 ### Prerequisites
 
-* Read permission to the target GitHub repository `pull-requests`.
+* Write permission to the target GitHub repository `pull-requests`.
```

Comment:
Feels strange that these two commands are the only ones that call out the github token permissions in the body text, but the rest dont. Ideally we should add the same for all commands or remove these since they are documented up top.

Reply:
Or link back to the top when referencing permissions, maintaining in two places might not be ideal.

Reply:
went with linking back to the permissions table.